### DISCUSSION TITLE
适配Vuepress的代码片段

### DISCUSSION THREAD

**Discussion by zxiaosi** [walinejs/waline#246] [Q&A] [answered]
大佬，能写一个适配vuepress的教程吗，我试了代码片段加不上去

**Comment by lizheming** [comment]
https://github.com/lizheming/waline/blob/master/docs/.vuepress/appEnhance.js 参考一下官网的配置？

**Comment by Mister-Hope** [comment, marked as answer]
mr-hope/vuepress-plugin-comment
